### PR TITLE
Title updates

### PR DESCRIPTION
**Pull request by amkuchta**
Updates check titles to address the first global task in #174

### CODE REVIEW

**Comment by amkuchta** [comment]
FYSA, these titles were `ctrl+c`/`ctrl+v` right from the v2r2 STIG - if there is an error, let me know, but everything should line up.  Also, ignore the first CAT1 title update - it reformatted the whole file, which would have been a pain to diff.  I reverted and resubmitted in the last commit.

File: tasks/audit_system_call.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 # tasks/audit_system_call.yml
 
-- name: "MEDIUM | RHEL-07-{{ item.id }} | {{ audit_present | ternary('PATCH', 'REVERT') }} | All uses of the {{ item.call }} command must be audited."
+- name: "MEDIUM | RHEL-07-{{ item.id }} | {{ audit_present | ternary('PATCH', 'REVERT') }} | The Red Hat Enterprise Linux operating system must audit all uses of the  {{ item.call }} syscall."
```
Comment (jamescassell) [inline tasks/audit_system_call.yml]: Are there two spaces in a row in the DISA content?

Comment (amkuchta) [reply tasks/audit_system_call.yml]: Talk about "eagle-eyed"...

I just pushed an update to correct the extra space!  Should be good to go!

**Comment by jamescassell** [comment]
@amkuchta Do you mind if I rebase out the reverted commit?  Otherwise, I'd recommend squashing to merge this PR.

**Comment by amkuchta** [comment]
@jamescassell nope, feel free to do as you see necessary!

**Pull request merged**